### DISCUSSION TITLE
x86 (32 bit) Windows Compability

### DISCUSSION THREAD

**Discussion by inlophe** [netbirdio/netbird#473] [Issue Triage]
Hi, is it possible to add  32bit windows release? I have been trying netbird itself and it's fantastic, but I have several old 32bit windows pc that I want to include in the network. Is this possible?

**Comment by braginini** [comment]
Hey @inlophe 

I think this should be no problem. We will check and get back to you.

**Comment by inlophe** [comment]
Any update on this? Is it possible?

**Reply by mlsmaycon** [reply]
hello @inlophe can you tell us which version of windows are you running? We would like to test it before adding support

**Reply by inlophe** [reply]
Windows 7 Professional 32bit, Can't really upgrade several computers with this to 64bit.

**Reply by mlsmaycon** [reply]
Thanks, I will to test it and get back to you

**Reply by jlq76** [reply]
32bits support woud be great! 
Running win10 x32 on an old Asus T100TA  which cannot be upgraded to x64 (uefi limitation).

**Comment by inlophe** [comment]
Any update? Or problems
